src/SourceMapToolkit.CallstackDeminifier/StackFrameDeminifier.cs

```diff
@@ -1,87 +1,91 @@
-﻿using System;
```

Comment:
Could you please take a look at the whitespace diff in this file and see what's wrong? Maybe line endings? It would be great to not mess up the history of other lines.

Reply:
Yeah my bad.  I tried to remove the extra indent in the Web UI and I think they handle \r\n differently so everything got mangled.  I'm trying to fix it but having trouble synching to that branch.

Reply:
@ian-craig  - should be fixed now.